roles/kerberos5/templates/hosts.j2

```diff
@@ -0,0 +1,3 @@
+127.0.0.1	localhost.localdomain localhost
+{{ krb5_master_address }}	{{ inventory_hostname_short | lower }}.{{ krb5_realm | lower }} {{ inventory_hostname_short | lower }}
```

Comment:
Here you convert `krb5_realm` to lower case, in other places to upper case, and in more places it's just used as is. Should it be upper case or lower case in the stack parameters, after all?

Reply:
We're using krb5_realm as both hostname and realm name but hostnames traditionally mentioned in lowercase and realm names are working only when written in uppercase.

Reply:
So,

> Should it be upper case or lower case in the stack parameters, after all?

Reply:
It does not matter. I'll plan to just consistently use `lower` and `upper` filters where needed.

Reply:
Well, I was asking because now there are at least 5 places in this PR where you use `krb5_realm` without case conversion. Now I now they should be fixed. Please do.

Reply:
Fixed.